shared/js/background/chrome-events.es6.js

```diff
@@ -175,6 +175,24 @@ function blockingExperimentActive () {
     // return false
 }
 
+// we determine if FLoC is enabled by testing for availability of its JS API
+const isFlocEnabled = ('interestCohort' in document)
+
+// Overwrite FLoC JS API
+if (isFlocEnabled) {
+    chrome.webNavigation.onCommitted.addListener(details => {
+        const tab = tabManager.get({ tabId: details.tabId })
+        if (tab && tab.site.whitelisted) return
+
+        chrome.tabs.executeScript(details.tabId, {
+            file: 'public/js/content-scripts/floc.js',
+            allFrames: true,
```

Comment:
See https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/523. Confusingly, `onCommitted` is called when any frame changes location, not just the main frame.

Reply:
Thanks Dave, good catch! I addressed it.